aqua/core/console/analysis.py

```diff
@@ -11,6 +11,7 @@
 from aqua.core.configurer import ConfigPath
 from aqua.core.util import expand_env_vars
 from aqua.core.logger import log_configure
+from aqua import __path__ as aqua_pkg_path
```

Comment:
we are now using importlib.resources for this operations. Please check how this is done in the `main` of the console